libs/gltfio/src/TangentsJob.cpp

```diff
@@ -49,7 +49,7 @@ void TangentsJob::run(Params* params) {
     std::unique_ptr<float3[]> morphDeltas;
 
     // Build a mapping from cgltf_attribute_type to cgltf_accessor.
-    const int NUM_ATTRIBUTES = 8;
+    const int NUM_ATTRIBUTES = cgltf_attribute_type_max_enum;
```

Comment:
Should it be `std::min(cgltf_attribute_type_max_enum, 8)` or have some assert if it's more than 8? @poweifeng ?